### PR TITLE
Use new Nuxt SDK in the init

### PR DESCRIPTION
**Pull request by Miniplop**
Use now the new nuxt SDK

### CODE REVIEW

File: packages/core/src/models/Framework.ts [hunk 1]
```diff
@@ -14,6 +15,7 @@ export enum Frameworks {
 export const FrameworksC = t.keyof({
   [Frameworks.none]: null,
   [Frameworks.nuxt]: null,
+  [Frameworks.previousNuxt]: null,
```
Comment (MarcMcIntosh) [inline packages/core/src/models/Framework.ts:18]: .... how will this be written in the sm.json ?

File: packages/slice-machine/postinstall.js [hunk 2]
```diff
@@ -56,29 +51,10 @@ function writeSMVersion(smModuleCWD, smConfig) {
...
-function installSMScript(pkg) {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-  if (!pkg.value.scripts) {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    pkg.value.scripts = {};
-  }
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-  if (!pkg.value.scripts.slicemachine) {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    pkg.value.scripts.slicemachine = "start-slicemachine --port 9999";
```
Comment (MarcMcIntosh) [inline packages/slice-machine/postinstall.js:68]: where is this script added?

**Review by MarcMcIntosh — COMMENTED** [review]
You might want to check this file too
`packages/slice-machine/server/src/api/simulator.ts`

File: packages/slice-machine/postinstall.js (hunk 2 above)
Comment (Miniplop) [reply packages/slice-machine/postinstall.js:68]: in the init script with FileSystem.addJsonPackageSmScript(cwd);

File: packages/core/src/models/Framework.ts (hunk 1 above)
Comment (Miniplop) [reply packages/core/src/models/Framework.ts:18]: with a migration that will update the sm.json with a framework : "previous-nuxt"

**Comment by Miniplop** [comment]
For the simulator it's part of an another ticket, so I will make the modification on the next tasks

File: packages/slice-machine/postinstall.js (hunk 2 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/postinstall.js:68]: neat

File: packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/index.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { Frameworks } from "@slicemachine/core/build/src/models/Framework";
 
 const FrameworkRenderers = {
   [Frameworks.nuxt]: Renderers.nuxt,
+  [Frameworks.previousNuxt]: null,
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/index.tsx:10]: Are the hints in another tickets?

**Review by MarcMcIntosh — APPROVED** [review]
Are the hints in another ticket?

File: packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/index.tsx (hunk 3 above)
Comment (Miniplop) [reply packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/index.tsx:10]: Yes here https://github.com/prismicio/slice-machine/pull/369

**Pull request merged**